python/ray/autoscaler/v2/tests/test_sdk.py

```diff
@@ -753,7 +753,7 @@ def verify_nodes_busy():
     stub = _autoscaler_state_service_stub()
     state = autoscaler_pb2.AutoscalingState(
         last_seen_cluster_resource_state_version=0,
-        autoscaler_state_version=1,
+        autoscaler_state_version=10,  # use a large number to avoid flaky.
```

Comment:
would you mind adding more context in the comment?

Reply:
Done.